### PR TITLE
Add `--help` and `--version` command line arguments.

### PR DESCRIPTION
**Pull request by supersam654**
As requested in #1212, it would be really nice if Hyper recognized the `--help` command line argument. 

Seeing as Hyper doesn't support any actual arguments, I opted to just print the version, location of the config file, and a brief message explaining that there are no command line arguments.

It felt weird supporting `--help` without `--version` so `--version` and `-v` do the same thing.

I've been trying to upload a screenshot but Github isn't cooperating right now so this will have to do:

```
$ ./Hyper.exe --help

Hyper version 1.0.1
Hyper does not accept any command line arguments. Please modify the config file instead.
Hyper configuration file located at: C:\Users\Sam\.hyper.js
```

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -1,3 +1,15 @@
+const firstArg = process.argv[1];
+// Print diagnostic information for a few arguments instead of running Hyper.
+if (firstArg === '--help' || firstArg === '-v' || firstArg === '--version') {
```
Comment (anonrig) [inline app/index.js]: What about?

```['--help', '-v', '--version'].indexOf(firstArg) != -1```

File: app/index.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
+// Print diagnostic information for a few arguments instead of running Hyper.
+if (['--help', '-v', '--version'].indexOf(process.argv[1]) !== -1) {
+  const {version} = require('./package');
+  const configLocation = process.platform === 'win32' ? process.env.userprofile + '\\.hyper.js' : '~/.hyper.js';
+  console.log('Hyper version ' + version);
```
Comment (albinekb) [inline app/index.js]: use template-strings
```js
console.log(`Hyper version  ${version}`)

File: app/index.js [hunk 3]
```diff
@@ -1,3 +1,14 @@
+// Print diagnostic information for a few arguments instead of running Hyper.
+if (['--help', '-v', '--version'].indexOf(process.argv[1]) !== -1) {
```
Comment (albinekb) [inline app/index.js]: use [.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) instead 🎉

**Comment by matheuss** [comment]
Neat, thank you @supersam654 💯

**Pull request merged**